code/modules/events/infestation.dm

```diff
@@ -19,7 +19,6 @@
 		INFESTATION_MICE = 1,
 		INFESTATION_LIZARDS = 1
 	)
-	no_fake = 1
```

Comment:
Oh wait, put this back, because it would give an empty feedback when it was fake or something.